src/modules/home/components/LatestTransactionDataTable.vue

```diff
@@ -166,7 +173,7 @@ export default{
       isFetching.value = false;
     }
 
-    setInterval(generateDatatable, 60000);
+    setInterval(generateDatatable, 15000);
```

Comment:
set refresh interval to match network config `blockGenerationTargetTime`

also, as a feature: disable refresh by default unless user select "live"